dynawo/3rdParty/xercesc/patch/MSVC/xercesc.patch

```diff
@@ -14,19 +15,3 @@ diff -urpN xerces-c-3.2.2/CMakeLists.txt xerces-c-3.2.2.patch/CMakeLists.txt
  # Use new variable expansion policy.
```

Comment:
same here for the policy and I think the patch can go away

Reply:
OK